galaxy_ng/app/api/ui/urls.py

```diff
@@ -156,12 +156,12 @@
     path('controllers/', views.ControllerListView.as_view(), name='controllers'),
     path('groups/', include(group_paths)),
     path(
-        'repo/<str:path>/',
+        'repo/<str:distro_base_path>/',
```

Comment:
I see this change reflected in the pulp_ansible pr with no further hints there.
What is the final goal? Should it eventually be called `distro_base_path` in all places?